src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/Util.cs

```diff
@@ -103,7 +119,7 @@ public static X509Certificate2 CertificateFromSubject(StoreName name, StoreLocat
             }
         }
 
-        public static X509Certificate2 CertificateFromFridendlyName(StoreName name, StoreLocation location, string fridendlyName)
+        public static X509Certificate2 CertificateFromFriendlyName(StoreName name, StoreLocation location, string friendlyName)
```

Comment:
Reviewers: nit level and not required for this product feature, but these misspellings were constantly fooling my searches for "friendlyName", so I fixed them.